### ISSUE TITLE
kubernetes e2e test failures

### ISSUE THREAD

**Issue by lwr20** [projectcalico/calicoctl#1554]
Running the NetworkPolicy tests from https://github.com/djosborne/kubernetes/tree/e2e-network-policy
I'm finding that 2 of the tests consistently (5/5) fail when using the kubernetes dataplane driver:
```
Summarizing 2 Failures:

[Fail] [k8s.io] NetworkPolicy [It] should enforce policy based on Ports [Feature:NetworkPolicy] 
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/network_policy.go:556

[Fail] [k8s.io] NetworkPolicy [It] should enforce multiple, stacked policies with overlapping podSelectors [Feature:NetworkPolicy] 
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/network_policy.go:556

Ran 7 of 574 Specs in 459.768 seconds
FAIL! -- 5 Passed | 2 Failed | 0 Pending | 567 Skipped --- FAIL: TestE2E (459.79s)
FAIL
```

**Comment by lwr20** [comment]
Possibly connected to https://github.com/projectcalico/felix/issues/1375

**Comment by lwr20** [comment]
Seems to be fixed by projectcalico/felix#1375
